modules/compliance-operator-rosa-installation.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-management/compliance-operator-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-compliance-operator-rosa_{context}"]
+= Installing the Compliance Operator on ROSA hosted control planes (HCP)
+
+As of the Compliance Operator 1.5.0 release, the Operator is tested against {product-rosa} using {hcp-capital}.
+
+{product-rosa} {hcp-capital} clusters have restricted access to the control plane, which is managed by Red Hat. By default, the Compliance Operator will schedule to nodes within the `master` node pool, which is not available in {product-rosa} {hcp-capital} installations. This requires you to configure the `Subscription` object in a way that allows the Operator to schedule on available node pools. This step is necessary for a successful installation on {product-rosa} {hcp-capital} clusters.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Define a `Namespace` object:
++
+.Example `namespace-object.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged <1>
+  name: openshift-compliance
+----
+<1> in {product-title} {product-version}, the pod security label must be set to `privileged` at the namespace level.
```

Comment:
I think you still need to apply Alex's comment in the other PR to capitalize this sentence:
```suggestion
<1> In {product-title} {product-version}, the pod security label must be set to `privileged` at the namespace level.
```